ush/sub_jet

```diff
@@ -98,34 +98,28 @@ task_node=${task_node:-$procs}
 size=$((nodes*task_node))
 envars=$envars
 threads=${rcpu:-1}
-#envars=$envars,mpi_tasks=$procs
-#Options
-###PBS -l partition=c1ms,size=0528,walltime=01:20:00
-##PBS -l partition=$queue,size=$size,walltime=$timew
-##PBS -S /bin/sh
 
 export TZ=GMT
 cfile=$DATA/sub$$
 > $cfile
-#echo "#PBS -S /bin/sh"                     >> $cfile
 echo "#!/bin/sh --login"                                                           >> $cfile
 echo ""                                                                            >> $cfile
 echo "#SBATCH --output=$output"                                                    >> $cfile
 echo "#SBATCH --job-name=$jobname"                                                 >> $cfile
 echo "#SBATCH --qos=$queue"                                                        >> $cfile
 echo "#SBATCH --time=$timew"                                                       >> $cfile
 echo "#SBATCH --nodes=$nodes --ntasks-per-node=$procs --cpus-per-task=$threads"    >> $cfile
-#echo "#SBATCH -j oe"                                       >> $cfile
 echo "#SBATCH --account=$accnt"                                                    >> $cfile
 echo "#SBATCH --mem=0"                                                             >> $cfile
-echo "#SBATCH --partition=kjet"                                                    >> $cfile
-#echo "#SBATCH -V"                                          >> $cfile
-#echo "#PBS -d"                             >> $cfile
-#. $exec                                    >> $cfile
-#echo "/bin/sh -x $exec"                    >> $cfile
+echo "#SBATCH --partition=xjet"                                                    >> $cfile
```

Comment:
I think this should stay as `kjet` by default.  It is the most commonly used partition for the global workflow, at least, and possibly for other applications using the GSI.